site/pages/add-consequence.test.tsx

```diff
@@ -0,0 +1,30 @@
+import renderer from "react-test-renderer";
+import { describe, it, expect } from "vitest";
+import AddConsequence, { AddConsequenceProps, TransportMode, ConsequenceType } from "./add-consequence";
+import { ErrorInfo } from "../interfaces/index";
+
+const noErrors: ErrorInfo[] = [];
+const withNoInputs: AddConsequenceProps = {};
+
+const withErrors: ErrorInfo[] = [
+    { id: "consequence-type-services", errorMessage: "Please select a consequence type" },
+    { id: "transport-mode-bus", errorMessage: "Please select a mode of transport" },
```

Comment:
Error messages need to never have the word 'Please' in them

Reply:
done